tests/container_checker/test_container_checker.py

```diff
@@ -203,18 +204,26 @@ def test_container_checker(duthosts, enum_rand_one_per_hwsku_hostname,
     container_name = asic.get_docker_name(service_name)
 
     loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="container_checker_{}".format(container_name))
-
+    sleep_time = 70
     disabled_containers = get_disabled_container_list(duthost)
 
     skip_containers = disabled_containers[:]
 
     # Skip 'radv' container on devices whose role is not T0/M0.
     if tbinfo["topo"]["type"] not in ["t0", "m0"]:
         skip_containers.append("radv")
-
     pytest_require(service_name not in skip_containers,
                    "Container '{}' is skipped for testing.".format(container_name))
-
+    feature_autorestart_states = duthost.get_container_autorestart_states()
+    if feature_autorestart_states.get(service_name) == 'enabled':
+        disable_container_autorestart(duthost)
+        time.sleep(30)
+    if not is_container_running(duthost, container_name):
+        logger.info("Container '{}' is not running ...".format(container_name))
+        logger.info("Reload config on DuT as Container is not up '{}' ...".format(duthost.hostname))
+        config_reload(duthost, safe_reload=True)
+        time.sleep(300)
+        sleep_time = 80
```

Comment:
why do we need to modify the sleep_time to 80 if the container is not running, given that there is alrady sleep(300) ?

Reply:
@SuvarnaMeenakshi 
 The Particular Sleep time is set for the log Analyzer, which is 70s in case the container was in start state, and 80s in case of config reload. A Increase of 10s is added to ensure logging in syslogs for the latter case. 
![image](https://github.com/sonic-net/sonic-mgmt/assets/135994174/1085a8d1-dd93-4af1-a36c-e3e5a8f3ff63)

Reply:
@SuvarnaMeenakshi , After the config reload the timer of container checker get reset, and issue is observed , as per the /etc/monit/conf.d/sonic-host the image attached for reference, A 50 Second delay is observed between the checks when no issue is observed, which and the daemon is set for 10s with start delay of 10s, So in order to escape the corner case an extra wait of 10s is added.
![image](https://github.com/sonic-net/sonic-mgmt/assets/135994174/38094259-aebc-46f7-96cb-a5e35000eea1)

Reply:
ok thank you